docs/visual-basic/programming-guide/language-features/xml/index.md

```diff
@@ -1,25 +1,21 @@
 ---
 title: "XML in Visual Basic"
-ms.custom: ""
-ms.date: 07/20/2015
+description: "Learn how to use LINQ to XML features in your Visual Basic code."
```

Comment:
Not all the remaining topics cover LINQ to XML. Perhaps, "Learn how to use XML and LINQ to XML features in your Visual Basic code."?